src/chapters/how-to-code/what-is-code.rst

```diff
@@ -6,7 +6,7 @@ actually dumb. They react mechanically to the instructions we give them, and
 they cannot deviate from the steps we tell them to follow.
 
 If our instructions are even the slightest bit off, computers cannot ignore
-the error and carry on. Instead, they come to a grinding halt.
+the error and carry on. Instead, they may come to a grinding halt.
```

Comment:
Logic errors aside, I prefer the more concrete result over the "may" path at this early stage.  Thoughts?

Reply:
I added the "may" because of logic errors. I wasn't sure how productive it was to have a sentence stating that computers come to a grinding halt with any error to then later refute it when talking about logic errors. I think a more concrete result in this instance could lead to confusion at a later point.